package.json

```diff
@@ -31,7 +31,8 @@
     "@datorama/akita": "^6.1.3",
     "@datorama/akita-ng-router-store": "^6.0.0",
     "akita-ng-fire": "^6.0.0",
-    "firebase": "^8.0.1",
+    "firebase": "^9.1.0",
+    "rxfire": "^6.0.2",
```

Comment:
is rxfire a perrDependancy of @angular/fire ?

Reply:
It is. `ng add @angular/fire` automatically add it as will `ng update @angular/fire`.